src/views/components/index.ts

```diff
@@ -0,0 +1,4 @@
+(function () {
+  const globalsContext = require.context('./global', true, /index\.vue/);
+  globalsContext.keys().forEach(globalsContext);
+}());
```

Comment:
Import ALL the global components, as this is all that needs to be done to use them